src/main/java/Duke/Event.java

```diff
@@ -0,0 +1,28 @@
+package Duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a task that will occur on a particular date
+ */
+
+public class Event extends Task {
+
+    private LocalDate at;
+
+    public Event(String description, LocalDate at) {
+        super(description);
+        this.at = at;
+    }
+
+    /**
+     * Returns the string representation of the event
+     * @return String representation of the task.

```

Comment:
There should be a line spacing between the parameters and the description as well!